### PR TITLE
initialVisibilityDelay added to PutRawMessageAsync and PutMessageAsync

### PR DESCRIPTION
**Pull request by PaterSantyago**

### CODE REVIEW

File: src/Lykke.AzureStorage/Queue/AzureQueueExt.cs [hunk 1]
```diff
@@ -107,16 +119,32 @@ public async Task FinishMessageAsync(QueueData token)
...
+        {
+            return InnerPutMessageAsync(itm, null);
+        }
+        
+        public Task<string> PutMessageAsync(object itm, TimeSpan initialVisibilityDelay)
+        {
+            return InnerPutMessageAsync(itm, initialVisibilityDelay);
+        }
+        
+        public async Task<string> InnerPutMessageAsync(object itm, TimeSpan? initialVisibilityDelay)
```
Comment (starkmsu) [inline src/Lykke.AzureStorage/Queue/AzureQueueExt.cs]: This method must be private - acording to its name

File: src/Lykke.AzureStorage/Queue/AzureQueueExt.cs [hunk 2]
```diff
@@ -107,16 +119,32 @@ public async Task FinishMessageAsync(QueueData token)
...
         {
             var msg = SerializeObject(itm);
+            
             if (msg == null)
```
Comment (starkmsu) [inline src/Lykke.AzureStorage/Queue/AzureQueueExt.cs]: string.NullOrWhiteSpace?

File: src/Lykke.AzureStorage/Queue/IQueueExt.cs [hunk 3]
```diff
@@ -31,7 +31,9 @@ public interface IQueueExt
         string Name { get; }
 
         Task PutRawMessageAsync(string msg);
+        Task PutRawMessageAsync(string msg, TimeSpan initialVisibilityDelay);
```
Comment (starkmsu) [inline src/Lykke.AzureStorage/Queue/IQueueExt.cs]: Let's add xml doc to new methods

File: src/Lykke.AzureStorage/Queue/AzureQueueExt.cs (hunk 2 above)
Comment (PaterSantyago) [reply src/Lykke.AzureStorage/Queue/AzureQueueExt.cs]: Mad exactly as it has been done earlier. In fact, queue message with empty body is ok.

Comment (starkmsu) [reply src/Lykke.AzureStorage/Queue/AzureQueueExt.cs]: And why do we need to skipp null body?

Comment (starkmsu) [reply src/Lykke.AzureStorage/Queue/AzureQueueExt.cs]: I see that in prev implementation null is also skipped - so this is for compatibility reasons. Agree with curretn version.

**Pull request merged**